### ISSUE TITLE
Terraform Missing Permissions for PKS Deploy on AWS

### ISSUE THREAD

**Issue by mcowger** [pivotal-cf/docs-pks#381]
After adding the requested permissions:
```
AmazonEC2FullAccess
AmazonRDSFullAccess
AmazonRoute53FullAccess
AmazonS3FullAccess
AmazonVPCFullAccess
IAMFullAccess
```

 terraform was unable to deploy fully due to missing permissions on kms:TagResource:
```
Error: Error applying plan:

1 error(s) occurred:

* aws_kms_key.blobstore_kms_key: 1 error(s) occurred:

* aws_kms_key.blobstore_kms_key: AccessDeniedException: User: arn:aws:iam::XXXXXXX:user/pksuser is not authorized to perform: kms:TagResource on resource: arn:aws:kms:us-west-2:XXXXX:key/00000000-0000-0000-0000-000000000000
        status code: 400, request id: 3a774a20-6b7e-47f5-a50a-715586f71eba
```

I ended up adding 'AdministratorAccess' temporarily, but we should figure out which was missing.

**Comment by bentarnoff** [comment]
Thanks @mcowger. I know that CF Infrastructure and OD-PKS are working on updating the AWS Terraform templates. (You can see the OD-PKS story at https://www.pivotaltracker.com/n/projects/2113002/stories/160412476). @outofmem0ry might have some additional context on the progress of that effort.

**Comment by bentarnoff** [comment]
Thanks, @mcowger. @outofmem0ry @neil-hickey can I ask you to look at the above and help determine which permission is missing? Then we can update the docs accordingly. Thanks.

**Comment by bentarnoff** [comment]
Hey @mcowger can I ask you what version of the Terraform templates you were using? We've been unable to reproduce this error in the AWS Terraform Templates v0.22.0 available under the [PAS v2.3.3 section in PivNet](https://network.pivotal.io/products/elastic-runtime). We've also made extensive revisions to the [PKS AWS docs](https://docs.pivotal.io/runtimes/pks/1-2/aws-index.html) to reflect the new Terraform process.

**Comment by bentarnoff** [comment]
@mcowger Closing this issue but please feel free to get in touch if you have further concerns or feedback.
